subscribers:
#  sensor_name:
#    channels: ['feat_0','feat_1']
#    fusion: ['average','average']
#    topic_name: '/elevation_mapping/pointcloud_semantic'
  front_cam:
    channels: ['rgb'] #'feat_0','feat_1','person','grass']
    fusion: ['color'] #'average','average','class_average','class_average']
    topic_name: '/elvation_mapping/pointcloud_semantic'
    semantic_segmentation: False
    segmentation_model: 'detectron_coco_panoptic_fpn_R_101_3x'
    publish_segmentation_image: True
    feature_extractor: False
    feature_config:
      name: 'DINO'
      interpolation: 'bilinear'
      model: "vit_small"
      patch_size:  16
      dim: 5
      dropout: False
      dino_feat_type: "feat"
      input_size: [80, 160]
      projection_type: "nonlinear"

    cam_info_topic: "/zed2i/zed_node/depth/camera_info"
    image_topic: "/zed2i/zed_node/left/image_rect_color"
    depth_topic: "/zed2i/zed_node/depth/depth_registered"
    cam_frame: "zed2i_right_camera_optical_frame"
    confidence_topic: "/zed2i/zed_node/confidence/confidence_map"
    confidence_threshold: 10


